projects/optic/src/commands/oas/diffing/document.ts

```diff
@@ -73,18 +75,19 @@ export async function addIfUndocumented(
 }
 
 async function computeOperationsToAdd(
-  input: string,
+  operationsToAdd: ParsedOperation[],
+  isAddAll: boolean,
```

Comment:
Instead of having isAddAll, why not just pass the entire list operations if `isAddAll` and the smaller list of operations.

This avoids having to pass down 2 laters of fn `addIfUndocumented` and `computeOperationsToAdd`

Reply:
We don't know the operations at the earlier stage